assets/js/apps/search/components/SimpleSearchForm.jsx

```diff
@@ -43,54 +43,40 @@ function SimpleSearchForm({ showResults, searchText }) {
     fetchResults();
   }, [searchText]);
   return (
-    <main>
-      <div className="row">
-        <div className="mx-auto align-items-center">
-          <div className="card align-items-center">
-            <div className="card-body">
-              <div className="row align-items-center">
-                <div className="col-md-12">
-                  <form className="form-horizontal" onSubmit={handleSimpleSearchSubmit} id="simple-search">
-                    <div className="input-group mb-3">
-                      <input
-                        type="text"
-                        name="simple_search_text"
-                        onChange={event => handleSimpleSearchTextChange(event, event.target.value)}
-                        value={simpleSearchText}
-                        className="form-control"
-                        placeholder="Search for Experiments, Datasets, Datafiles"
-                      />
-                      <div className="input-group-append">
-                        <button
-                          type="button"
-                          className="input-group-text"
-                          onClick={handleSimpleSearchSubmit}
-                        >
-                          <span className="fa fa-search" />
-                        </button>
-                      </div>
-                    </div>
-                    <AdvancedSearchForm
-                      searchText={simpleSearchText}
-                      showResults={showResults}
-                    />
-                  </form>
-                </div>
-              </div>
-            </div>
+    <div>
+      <form onSubmit={handleSimpleSearchSubmit} id="simple-search" className="my-3">
+        <div className="input-group">
+          <input
+            type="text"
+            name="simple_search_text"
+            onChange={event => handleSimpleSearchTextChange(event, event.target.value)}
+            value={simpleSearchText}
+            className="form-control"
+            placeholder="Search for Experiments, Datasets, Datafiles"
+          />
+          <div className="input-group-append">
+            <button
+              type="button"
+              className="btn btn-primary"
+              onClick={handleSimpleSearchSubmit}
+            >
+              <span className="fa fa-search" />
+            </button>
           </div>
         </div>
-      </div>
-      {isLoading
-       && (
-       <div className="col-md-12" style={{ textAlign: 'center', position: 'absolute' }}>
-         <div id="spinner" style={{ textAlign: 'center' }}>
-           <i id="mo-spin-icon" className="fa fa-spinner fa-pulse fa-2x" />
-         </div>
-       </div>
-       )
-      }
-    </main>
+      </form>
+      {isLoading && (
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 12 spaces but found 6.](https://app.codacy.com/gh/mytardis/mytardis/pullRequest?prid=7933941)
```suggestion
            {isLoading && (
```